requirements/optional-private.txt

```diff
@@ -3,7 +3,7 @@
 # git config --global url.https://<private hostname>/.insteadOf https://private.repo/
 
 git+https://private.repo/CFPB/agreement_database.git@2.2.7#egg=agreement-database
-git+https://private.repo/CFPB/cfgov-selfregistration.git@v1.2#egg=selfregistration
+selfregistration==@1.3.1
```

Comment:
yep! The build is already configured to use the repo, so this will be the first test